### DISCUSSION TITLE
Can't sync automatically

### DISCUSSION THREAD

**Discussion by pxd0207** [dani-garcia/vaultwarden#2913] [Q&A]
The password saved on the mobile phone cannot be automatically synchronized to other devices immediately.
![image](https://user-images.githubusercontent.com/55773646/201566497-fa649809-b535-47f1-9639-e5365757955a.png)
this is my config file

**Comment by pxd0207** [comment]
Need to manually click sync on other devices

**Comment by jjlin** [comment]
Vaultwarden doesn't currently support "live sync" for mobile clients. This is mentioned at https://github.com/dani-garcia/vaultwarden/wiki#missing-features.

**Reply by pxd0207** [reply]
I'm using chrome and win client and it can't sync either

**Reply by BlackDex** [reply]
Then double-check if you have configured the websockets correctly.
https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples

**Reply by pxd0207** [reply]
docker run -d --name bitwardenrs \
  --restart unless-stopped \
  -e WEBSOCKET_ENABLED=true \
  -v /www/wwwroot/flomoapp.me/:/data/ \
  -p 6666:80 \
  -p 3012:3012 \
  vaultwarden/server:latest

Here is the script I installed

**Reply by BlackDex** [reply]
Again, i would double check your reverse proxy config, see `Nginx (by blackdex)` you are having an invalid config. For one, you really need to add `proxy_http_version 1.1;` to the `/notifications/hub` config.

**Reply by BlackDex** [reply]
Also, live-sync from/to mobile client is not working. You need to configure sync on refresh in the mobile client, which will then sync when you pull-down to refresh.

**Reply by pxd0207** [reply]
location / {
            proxy_pass http://127.0.0.1:6666;
            proxy_http_version    1.1;
            proxy_cache_bypass    $http_upgrade;
            proxy_set_header Upgrade            $http_upgrade;
            proxy_set_header Connection         "upgrade";
            proxy_set_header Host               $host;
            proxy_set_header X-Real-IP          $remote_addr;
            proxy_set_header X-Forwarded-For    $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto  $scheme;
            proxy_set_header X-Forwarded-Host   $host;
            proxy_set_header X-Forwarded-Port   $server_port;
        }

        location /notifications/hub {
            proxy_pass http://127.0.0.1:3012;
            proxy_set_header Upgrade $http_upgrade;
            proxy_set_header Connection "upgrade";
        }

        location /notifications/hub/negotiate {
            proxy_pass http://127.0.0.1:6666;
        }

        location /admin {
            proxy_set_header Host $host;
            proxy_set_header X-Real-IP $remote_addr;
            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
            proxy_set_header X-Forwarded-Proto $scheme;
            proxy_pass http://127.0.0.1:6666;
        }
        # 加入robots.txt 防止搜索引擎爬虫抓取
        location = /robots.txt {
            root /home/wwwroot/Bitwarden;
        }
**Is this configuration file correct?**

**Reply by BlackDex** [reply]
No, it isn't. Look at the examples, it doesn't match as i mentioned in my previous post.

**Comment by omueller** [comment]
Watching too...   I still get the
```
[2022-10-31 00:29:01.094][vaultwarden::api::notifications][ERROR] 
    ###########################################################
    '/notifications/hub' should be proxied to the websocket server or notifications won't work.
    Go to the Wiki for more info, or disable WebSockets setting WEBSOCKET_ENABLED=false.
    ###########################################################################################

```
message in any case, also after trying (at least I think) all possible examples from the [wiki/Proxy-examples](https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples) page or several other discussions/tickets.    In my case it's an apache proxy, but currently wondering if the notifications part is working at all or for at least somebody else :)

I'm in the process of setting up a dev/test VM with current dev version to see if it helps.  

cheers, Olivier

**Reply by BlackDex** [reply]
If you follow the exact example `Apache (by fbartels)` including checking the needed extensions/mods/plugins to enable, that should work just fine as far as i know.

**Reply by pxd0207** [reply]
![image](https://user-images.githubusercontent.com/55773646/201929350-d217685e-f905-46ce-b053-9fd513c2cca5.png)

** how to do?**

**Reply by BlackDex** [reply]
> ![image](https://user-images.githubusercontent.com/55773646/201929350-d217685e-f905-46ce-b053-9fd513c2cca5.png)
> 
> ** how to do?**

You are using nginx, not Apache, so  you need to use `Nginx (by blackdex)` not the Apache example, that will never work.

**Reply by pxd0207** [reply]
thank u for ur time
